enterprise/backend/test/metabase_enterprise/sandbox/api/card_test.clj

```diff
@@ -3,68 +3,26 @@
    [clojure.test :refer :all]
    [metabase-enterprise.test :as met]
    [metabase.api.card-test :as api.card-test]
-   [metabase.models :refer [Card Collection Database PermissionsGroup
-                            PermissionsGroupMembership Table]]
-   [metabase.models.permissions :as perms]
-   [metabase.models.permissions-group :as perms-group]
    [metabase.query-processor :as qp]
    [metabase.test :as mt]
-   [metabase.util :as u]
    [toucan2.tools.with-temp :as t2.with-temp]))
 
-(deftest users-with-segmented-perms-test
```

Comment:
This test seems like it'd still be relevant, no?

Reply:
Good point, I was thinking that:
- this test was previously making sure that someone with segmented permissions could still save or update a query for a card. There's no GTAP involved, just the segmented perm.
- now, "segmented permissions" don't exist - the user has `:unrestricted` data permissions
- so the test would just be asserting that a user with `:unrestricted` data permissions can save or update the query for a card, not really testing the sandbox at all.

But it would probably make more sense to add a GTAP for the group and then make the same assertions - I'll do that.